.github/pull_request_template.md

```diff
@@ -0,0 +1,60 @@
+## Prerequisites for merge, code author responsibilities
+* [ ] I as the code author have ensured that acceptance criteria is met
+* [ ] I as the code author have self-reviewed code and functionality
+* [ ] I have added new tests/extended existing ones
+* [ ] If this contains frontend changes, a design review has been requested and completed
+* [ ] A security review is initialized according to [Security Chapter Policy](https://www.notion.so/Secure-code-review-75585ae5c1ea43dab08f3a390714afb5)
```

Comment:
We can skip this one as it is an internal document

Reply:
Fixed